source/fundamentals/databases-collections.txt

```diff
@@ -16,9 +16,9 @@ MongoDB organizes data into a hierachy of the following levels:
 - documents
 
 Databases are the top level of data organization in a MongoDB instance.
-Databases are organized into collections, which contain documents.
-Documents contain literal data like strings, numbers, and dates as well
-as other (embedded) documents.
+Databases are organized into collections, which contain :manual:`documents
+<core/document/>`. Documents contain literal data such as strings,
```

Comment:
I'm still not sure about this.  @ccho-mongodb is this a reasonable enough description of a document?

Reply:
How about:

Databases are organized into *collections* which contain *documents*. Documents contain... For more information on document field types and structure, see the server manual (linked).

Based on limited user feedback from MongoDB employees, we should explain first and link to further details, explaining what the links are for.

Since it's driver documentation, I don't want to be too redundant with the explanation of what they are since they're described in detail in the server documentation and instead focus more on how to manage them with the driver. If you think we're not meeting the latter goal, we should address that.

The one nitpick I might have is that you (Nate) could more clearly identify documents as the structure that hold your data.

Reply:
This sounds good to me -- making these changes now.